### ISSUE TITLE
runtime/pprof: test fails intermittently on windows/amd64

### ISSUE THREAD

**Issue by mikioh** [golang/go#10232]
??? which test case? why testing.Main cannot capture the condition?
```
ok  	runtime	37.654s
?   	runtime/cgo	[no test files]
ok  	runtime/debug	0.155s
exit status 2147483651
FAIL	runtime/pprof	8.979s
?   	runtime/race	[no test files]
ok  	sort	0.190s
```
See http://build.golang.org/log/dfb6e28b125b9068674f188c0658155d89c924fb and https://storage.googleapis.com/go-build-log/b284db5e/windows-amd64-gce_39c49307.log

**Comment by alexbrainman** [comment]
Dup of #10231 ?

Alex

**Comment by mikioh** [comment]
Yup, merged into #10231.
